### DISCUSSION TITLE
Intermittent sound / "We must catchup x seconds!" in Logs

### DISCUSSION THREAD

**Discussion by unudelaradio** [AzuraCast/AzuraCast#7060] [Q&A]
Hello
Does anyone knows what usually causes intermittent sound, common spotted at the top of hour and related with this logs:

```
2024/04/15 23:04:45 [clock.main:2] We must catchup 10.16 seconds!
2024/04/15 23:04:46 [clock.main:2] We must catchup 9.90 seconds!
2024/04/15 23:04:47 [clock.main:2] We must catchup 9.86 seconds!
2024/04/15 23:04:48 [clock.main:2] We must catchup 9.59 seconds!
```

I'm using the latest Stable Version on one of my servers. It happens on a station that has only few playlists, files no longer than 3 or 4 minutes. 

Are these _intermittent sound situations_ linked/related to something that can be avoid?

Thank you!

**Comment by LordHelmchen666** [comment]
Seems you have high CPU usage. Check with `top` what processes running at this time.
